scripts/mod2_pcoa.py

```diff
@@ -0,0 +1,276 @@
+
+	# mapping file
+	mf = pd.read_csv(mapping_file, '\t', converters=defaultdict(str), index_col='#SampleID')
+	mf = mf.loc[o.site_ids]
+
+	# coloring:  
+	# 'HMP-FECAL', 'GG-FECAL', 'PGP-FECAL' -- light red
+	# 'AGP-FECAL' -- dark red
+	# 'HMP-ORAL', 'PGP-ORAL' -- light blue
+	# 'AGP-ORAL' -- dark blue
+	# 'HMP-SKIN', 'PGP-SKIN' -- light orange
+	# 'AGP-SKIN' -- dark orange
+
+	cat_colors = {'HMP-FECAL': '#CB3F34', 'GG-FECAL': '#CB3F34', 'PGP-FECAL': '#CB3F34', 
+				  'AGP-FECAL': '#74151C', 
+				  'HMP-ORAL': '#64B4C5', 'PGP-ORAL': '#64B4C5',
+				  'AGP-ORAL': '#1D3773', 
+				  'HMP-SKIN': '#EDEE8E', 'PGP-SKIN': '#EDEE8E',
+				  'AGP-SKIN': '#D96B2D'}
+
+	for sample in mf.index:
+		for cat, color in cat_colors.iteritems():
+			sub_coords = c_df[mf.TITLE_BODY_SITE == cat]
+
+			plt.scatter(sub_coords[0], sub_coords[1], color=color, edgecolor='k', lw=LINE_WIDTH, alpha=ALPHA)
+
+		plt.scatter(c_df.loc[sample][0], c_df.loc[sample][1],
+					color=cat_colors[mf.loc[sample]['TITLE_BODY_SITE']],
+					s=270, edgecolor='w')
+		plt.scatter(c_df.loc[sample][0], c_df.loc[sample][1],
+					color=cat_colors[mf.loc[sample]['TITLE_BODY_SITE']],
+					s=250, edgecolor='k')
+		plt.axis('off')
+		my_dpi = 72
+		plt.savefig(sample+'.pdf', figsize=(1000/my_dpi, 1000/my_dpi), dpi=my_dpi)
+		plt.close()
+
+@mod1_pcoa.command()
+@click.option('--coords', required=True, type=click.Path(resolve_path=True, readable=True, exists=True), help='Coordinates file')
+@click.option('--mapping_file', required=True, type=click.Path(resolve_path=True, readable=True, exists=True), help='Mapping file')
+def country(coords,mapping_file):
+	"""Generates as many figures as samples in the coordinates file (Figure 1)"""
+	o = OrdinationResults.from_file(coords)
+	x, y = o.site[:,0], o.site[:,1]
+
+	# coordinates
+	c_df = pd.DataFrame(o.site, o.site_ids)
+
+	# mapping file
+	mf = pd.read_csv(mapping_file, '\t', converters=defaultdict(str), index_col='#SampleID')
+	mf = mf.loc[o.site_ids]
+
+	# coloring:  
+	# Australia
+	# Belgium
+	# Canada
+	# China
+	# Finland
+	# France
+	# Germany
+	# Great Britain
+	# Ireland
+	# Japan
+	# Malawi - BLUE
+	# Netherlands
+	# New Zealand
+	# Norway
+	# Scotland
+	# Spain
+	# Switzerland
+	# Thailand
+	# United Arab Emirates
+	# United Kingdom - PINK
+	# United States of America
+	# Venezuela - YELLOW
+	# no_data
+
+	cat_colors = {'Australia': '#FF0000',
+				  'Belgium': '#FF0000',
+				  'Canada': '#FF0000',
+				  'China': '#FF0000',
+				  'Finland': '#FF0000',
+				  'France': '#FF0000',
+				  'Germany': '#FF0000',
+				  'Great Britain': '#FF0000',
+				  'Ireland': '#FF0000',
+				  'Japan': '#FF0000',
+				  'Malawi': '#0000FF',
+				  'Netherlands': '#FF0000',
+				  'New Zealand': '#FF0000',
+				  'Norway': '#FF0000',
+				  'Scotland': '#FF0000',
+				  'Spain': '#FF0000',
+				  'Switzerland': '#FF0000',
+				  'Thailand': '#FF0000',
+				  'United Arab Emirates': '#FF0000',
+				  'United Kingdom': '#FF66FF',
```

Comment:
In the new version it's marked Western and then country of participant is changed to Highlight. 



> On May 17, 2015, at 9:38 PM, Daniel McDonald <notifications@github.com> wrote:
> 
> In scripts/mod2_pcoa.py:
> 
> > +				  'Finland': '#FF0000',
> > +				  'France': '#FF0000',
> > +				  'Germany': '#FF0000',
> > +				  'Great Britain': '#FF0000',
> > +				  'Ireland': '#FF0000',
> > +				  'Japan': '#FF0000',
> > +				  'Malawi': '#0000FF',
> > +				  'Netherlands': '#FF0000',
> > +				  'New Zealand': '#FF0000',
> > +				  'Norway': '#FF0000',
> > +				  'Scotland': '#FF0000',
> > +				  'Spain': '#FF0000',
> > +				  'Switzerland': '#FF0000',
> > +				  'Thailand': '#FF0000',
> > +				  'United Arab Emirates': '#FF0000',
> > +				  'United Kingdom': '#FF66FF',
> only UK stands out? Does this hardcoded value need to change for AG?
> 
> —
> Reply to this email directly or view it on GitHub.
>